### PR TITLE
Feature/tailwind

### LINKED ISSUE #12: Tailwind
Applying style using tailwind CDN.

### PR DESCRIPTION
**Pull request by thomps0189**
Started Tailwind, set up flex, but still need to size boxes better.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,31 +4,36 @@
...
 
-        </form>
-    </section>
+            </form>
+        </section>
 
-    <section>
-        <h2>Itinerary</h2>
+     <section class="flex m-0">
+        <h2 class="">Itinerary</h2>
```
Comment (DuGBB) [inline index.html:36]: Did you mean to leave an empty call declaration?

File: index.html [hunk 2]
```diff
@@ -41,9 +46,14 @@ <h3>Lodging</h3>
         <ul id="lodging-list" class="list-container">
             <li class="list-items">list will go here. </li>
         </ul>
+        </section> 
     </section>
 
-    <footer>Made by The Trail Lad's! &copy2021</footer>
+    
+
+    <footer class="">Made by The Trail Lad's! &copy2021</footer>
```
Comment (DuGBB) [inline index.html]: Did you mean to leave an empty class declaration?

File: index.html (hunk 1 above)
Comment (DuGBB) [inline index.html]: Did you mean to leave an empty class declaration?(I'm thinking these are for a future  purpose.  If you leave a comment next time, I'll know not  to bother you :relaxed:

File: index.html [hunk 3]
```diff
@@ -4,31 +4,36 @@
...
-        <h1>Vacation Planner for National Parks</h1>
-        <p>This section will tell us about what this site does. </p>
+        <h1 class="bg-green-600 text-center text-4xl">Vacation Planner for National Parks</h1>
+        <p class="text-center bg-blue-200">This section will tell us about what this site does. </p>
     </header>
 
-    <section>
-        <form class="form" id="location">
+    <section class="flex container">
+       <section class="">
```
Comment (DuGBB) [inline index.html]: Did you mean to leave an empty class?

File: index.html (hunk 2 above)
Comment (thomps0189) [reply index.html]: I meant to delete it. I had left it blank on purpose when trying to figure out the flex boxes.

**Review by DuGBB — APPROVED** [review]
Looks Great, Angela!

**Pull request merged**